src/js/components/stock-movement-wizard/outbound/PickPage.jsx

```diff
@@ -128,18 +129,22 @@ const FIELDS = {
         defaultMessage: 'Qty picked',
         flexWidth: '0.7',
         attributes: {
-          formatValue: value => (value ? (value.toLocaleString('en-US')) : value),
+          formatValue: value => (
+            <div className={!value ? 'text-danger' : null}>
+              {value.toLocaleString('en-US')}
+            </div>
+          ),
         },
       },
       buttonEditPick: {
-        label: 'react.stockMovement.editPick.label',
-        defaultMessage: 'Edit pick',
+        label: 'react.stockMovement.pick.label',
```

Comment:
There is one unknown here. Initially I thought this should be changed to pick only when we trigger clear picklist, but let's leave it as is for now, and let's see what Manon will say about that part.

Reply:
I got confirmation, that your current version is correct. You can disregard the previous comment here.